ansible/config_sonic_basedon_testbed.yml

```diff
@@ -75,14 +75,29 @@
     delegate_to: localhost
 
   - name: find interface name mapping and individual interface speed if defined from dut
-    port_alias: hwsku="{{ hwsku }}"
+    port_alias:
+      hwsku: "{{ hwsku }}"
+      cardtype: "{{ cardtype | default('fixed') }}"
```

Comment:
We use 'card_type' as the host variable to distinguish between linecard and supervisor card. Can we change 'cardtype' to 'card_type'.